website/assets/styles/pages/articles/basic-webinar.less

```
@@ -0,0 +1,753 @@
+        font-family: Inter;
+        font-size: 20px;
+        font-style: normal;
+        font-weight: 800;
+        line-height: 120%;
+      }
+    }
+  }
+
+  @media (max-width: 375px) {
+
+  }
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)